Derivs/src/derivs.cxx

```diff
@@ -10,9 +10,29 @@ using namespace Loop;
 
 template <int CI, int CJ, int CK, typename T>
 CCTK_ATTRIBUTE_NOINLINE void
-calc_derivs(const vec<GF3D5<T>, dim> &dgf, const GridDescBaseDevice &grid,
-            const GF3D5<const T> &gf, const GF3D5layout layout,
-            const vect<T, dim> dx, const int deriv_order) {
+calc_copy(const GF3D5<T> &gf0, const GF3D5layout layout,
```

Comment:
The output variables have the suffix `0`, the input variables the suffix `1`. That's a bit weird. Can you use `0` for the input variables, and use no suffix for the output variables?

Reply:
I modify suffix as suggested.